tests/Routing/RoutingUrlGeneratorTest.php

```diff
@@ -128,7 +128,7 @@ public function testControllerRoutesWithADefaultNamespace()
         $route = new Illuminate\Routing\Route(['GET'], 'foo/bar', ['controller' => 'namespace\foo@bar']);
         $routes->add($route);
 
-        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => 'something\foo@bar']);
+        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => '\something\foo@bar']);
```

Comment:
When calling ```$url->action('foo@bar')``` it appends the namespace to match ```namespace\foo@bar```
If you add a ```\``` it skips adding the root namespace since you don't want it.
If ```controller => '\something\foo@bar'``` is set, and you try to use
```$url->action('\something\foo@bar')``` It should match, but since it trims off all of the ```\```, it doesn't match: ```something\foo@bar !=  \something\foo@bar```